integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -169,6 +199,21 @@ impl TestReshardingParameters {
         self.loop_action = loop_action;
         self
     }
+
+    fn shuffle_shard_assignment(mut self, shuffle: bool) -> Self {
+        self.shuffle_shard_assignment_for_chunk_producers = shuffle;
+        self
+    }
+
+    fn track_all_shards(mut self, track_all_shards: bool) -> Self {
+        self.track_all_shards = track_all_shards;
+        self
+    }
+
+    fn all_chunks_expected(mut self, all_chunks_expected: bool) -> Self {
+        self.all_chunks_expected = all_chunks_expected;
+        self
+    }
```

Comment:
Yeah that's why I suggested @Trisfald to have a Builder pattern here...

Anyway, that's probably fine as is, let's just drop bool arguments and call method only if we need to change the default: `shuffle_shard_assignment`, `single_shard_tracking`, `dont_assert_empty_chunks (?!)`. It's not like we want to call these methods twice, ever.